src/libraries/System.IO.Pipes.AccessControl/README.md

```diff
@@ -0,0 +1,15 @@
+# System.IO.Pipes.AccessControl
+This assembly no longer contains executable code, it contains the code for the reference assembly and the tests for the types formerly implemented.
+
+## Source
+* All types previously implemented on this library are now part of [System.IO.Pipes](../System.IO.Pipes/) library.
+
+* Some of the tests for types previously in this library are still in the [tests](tests/) subdirectory.
+
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
```

Comment:
If the assembly no longer contains code it seems weird that we want features/etc :smile:

Reply:
Seems like I copy-pasted it and forgot to rewrite that section.